mix.exs

```diff
@@ -20,7 +20,8 @@ defmodule OEmbed.Mixfile do
   # Type "mix help compile.app" for more information
   def application do
     [applications: [:httpoison, :exconstructor],
-     extra_applications: [:logger]]
+     extra_applications: [:logger],
+     env: [third_party_providers: []]]
```

Comment:
@slavone Do you care if I rename it to just `providers`?
That would be cleaner

Reply:
@r8 Sure, this makes sense